README.md

```diff
@@ -7,18 +7,21 @@ Hides sensitive strings that get embedded in your app's binary.
 
 
 ```
-USAGE: ssg [--environment-path <environment-path>] [--checksum-path <checksum-path>] <output-path>
+USAGE: ssg-command [--environment-path <environment-path>] [--checksum-path <checksum-path>] [--template-path <template-path>] [--verbose] [--prioritize-alternative-keys] <output-path>
 
 ARGUMENTS:
-  <output-path>           path to generated file
+  <output-path>           Path to generated file.
 
 OPTIONS:
   -e, --environment-path <environment-path>
-                          Path to alternative keys file (defaults to .env)
+                          Path to alternative keys file (defaults to .env). (default: .env)
```

Comment:
Nitpick: This looks redundant. Do you need both `(defaults to .env)` and `(default: .env)`?